assets/src/gaussholder.js

```diff
@@ -144,26 +144,22 @@ export default function () {
 	 * Render all placeholders on the page
 	 */
 	return function () {
-		let images = document.getElementsByTagName( 'img' );
-
-		for ( let i = images.length - 1; i >= 0; i-- ) {
-			let img = images[i];
-
-			// Ensure the blank GIF has loaded first
-			if ( img.complete ) {
-				handleElement( img );
-			} else {
-				img.onload = function () {
-					handleElement( this );
-				};
-			}
-		}
+		const images = document.getElementsByTagName( 'img' );
+
+		const options = {
+			rootMargin: '1200px'
+		};
 
-		loadLazily = images;
-		scrollHandler();
```

Comment:
You've removed the call to this function - can the function definition above be removed?

Reply:
Thanks! As I'm working from the vendor folder, the IDE isn't warning me about these things